### DISCUSSION TITLE
Header Image Support

### DISCUSSION THREAD

**Discussion by KeithIMyers** [squidfunk/mkdocs-material#5849] [Ideas]
Good Afternoon,
I would love to be able to migrate my personal blog to mkdocs-material (I pay for the insiders build) but there  is one thing holding me back. I would love to be able to set a header image in each post to replicate the look and feel of my current website.

**Comment by squidfunk** [comment]
Thanks for suggesting. Could you provide more context? You can open a [change request](https://squidfunk.github.io/mkdocs-material/contributing/requesting-a-change/) – we need more input, screenshots, links to sites implementing what you describe, how you imagine this to work from an author's perspective, etc. ☺️ We're currently revamping the blog plugin for the initial 9.2 release and are happy to receive feedback, so we can make it even better afterwards!
